src/config.c

```diff
@@ -117,6 +117,12 @@ configEnum tls_auth_clients_enum[] = {
     {"optional", TLS_CLIENT_AUTH_OPTIONAL},
     {NULL, 0}};
 
+configEnum tls_client_auth_user_enum[] = {
+    {"CN", TLS_CLIENT_FIELD_CN},
+    {"O", TLS_CLIENT_FIELD_O},
```

Comment:
Latest from the weekly meeting is leave this as an enum but only support `CN` and `off` initially. We can add `O` or something else later as needed.